src/test/java/seedu/realodex/model/person/BirthdayTest.java

```diff
@@ -172,14 +172,27 @@ public void getDaysUntilBirthdayWithRepresentation_birthdayPresent_returnsDaysRe
     public void getDaysUntilBirthdayWithRepresentation_birthdayNotPresent_returnsDefaultString() {
         // Create Birthday object with birthday not present
         Birthday birthday = new Birthday();
-        birthday.setBirthday(Optional.empty());
+        birthday.setOptionalBirthday(Optional.empty());
 
         // Test
         assertEquals("Birthday is unspecified!", birthday.getDaysUntilBirthdayWithRepresentation());
```

Comment:
Guess all of this can remove `setOptionalBirthday` and just pass in the formatted string directly